### DISCUSSION TITLE
Using Shesa-framework, is the frontend only for React?

### DISCUSSION THREAD

**Discussion by Duelbat** [shesha-io/shesha-framework#979] [Ask for help]
Hi,
I use Blazor 100% for frontend and backend. You mention React for the frontend, I only use Blazor. Can this framework be used on the front end with Blazor?
Thanks

**Comment by teboho** [comment]
Technically you should be able to use Blazor (by connecting it with the backend that the framework provides)... but it defeats the purpose. 

React frontend is what shesha provides and is trying to enforce.
@OmolemoBlessingLethuloe can you speak to this?
